JSDemos/Demos/Switch/Overview/description.md

```diff
@@ -1 +1,6 @@
-The Switch is&nbsp;a&nbsp;UI component that can be&nbsp;in&nbsp;two states: &laquo;On&raquo; (when **value** is _true_) and &laquo;Off&raquo; (when **value** is _false_).
\ No newline at end of file
+The Switch component can be in two states: **ON** (when the [value](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#value) is **true**) and **OFF** (when the [value](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#value) is **false**). If you want to change the Switch text, specify the [switchedOnText](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#switchedOnText) and the [switchedOffText](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#switchedOffText) properties. If the changed text does not fit in the component, use the [width](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#width) property.
+
+To process a new Switch [value](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#value), you need to handle the value change event. Assign the handling function to the [onValueChanged](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#onValueChanged) property.
```

Comment:
I wouldn't use the phrase "to process a new Switch value". Looks like it implies that a developer MUST process values somehow. I would just say "to respond to value changes, assign a handling function to onValueChanged".

I would also change the order a little bit. First you say about **value** that can be **true** or **false**. In the same paragraph, you can talk about **onValueChanged** (related information). And then I would have information about **width**, **switchedOnText**, **swithcedOffText** in their own separate paragraph.

Reply:
Noted